fuzzers/nautilus_sync/src/lib.rs

```diff
@@ -120,13 +120,14 @@ pub extern "C" fn libafl_main() {
     let context = NautilusContext::from_file(15, "grammar.json");
 
     let mut event_converter = opt.bytes_broker_port.map(|port| {
-        LlmpEventConverter::on_port(
-            shmem_provider.clone(),
-            port,
-            Some(NautilusToBytesInputConverter::new(&context)),
-            none_input_converter!(),
-        )
-        .unwrap()
+        LlmpEventConverterBuilder::new()
```

Comment:
Launcher uses `LlmpEventConverter::builder()`

Reply:
`typed_builder` in general uses `::builder()`, we may want to keep this consistent (?)

Reply:
I guess `Builder::new()` is the pattern for non-typed-builder builders.
We could add a `::builder` method to the base struct anyway in this case

Reply:
in this way?
```
fn builder() -> Self {
 Self::new()
}
```

Reply:
maybe in other pr. now all libafl builder use new(), and not builder(), for its constructor

Reply:
They don't - all typed_builders use `::builder()` (and there's quite a few of them)